css/index.css

```diff
@@ -1,3 +1,552 @@
+  padding-left: 18px;
+}
+
+/* Footer */
+
+footer {
+  padding-top: 38px;
+  display: flex;
+  flex-direction: column;
+  height: 235px;
+}
+
+footer span {
+  display: block;
+  margin-bottom: 2px;
+}
+
+.span-big-margin {
+  margin-top: 20px;
+}
+
+.copyright {
+  padding-top: 50px;
+  align-self: center;
+  font-size: 14px;
+}
+
+/***************800PX BREAK***********/
+
+@media (max-width: 800px) {
+  /*General*/
+  .container {
+    padding: 15px 0;
+    max-width: 760px;
+  }
+
+  /*Nav Area*/
+  .sticky-nav {
+    flex-direction: column-reverse;
+    clip-path: none;
+    margin: 0 auto;
+  }
+
+  .sticky-nav img {
+    padding: 4px 0 0 15px;
+  }
+
+  .sticky-nav a {
+    align-self: center;
+    font-size: 16px;
+  }
+
+  .sticky-nav nav ul {
+    margin: 40px auto;
+    padding: 0 35px 0 20px;
+    justify-content: space-between;
+  }
+
+  .nav-ul li:not(:first-child) {
+    margin-left: 0;
+  }
+
+  /*Header Area*/
+  .header-left {
+    padding: 0;
+  }
+
+  .header-right {
+    display: none;
+  }
+
+  .main-header {
+    justify-content: center;
+    margin: 35px 0 0 0;
+    padding: 0 0 0 16px;
+    height: 345px;
+  }
+  .main-header h1 {
+    letter-spacing: 2px;
+    font-size: 70px;
+    margin-bottom: 30px;
+  }
+
+  /*Main*/
+  main {
+    padding-left: 10px;
+    padding-bottom: 0;
+  }
+
+  main section {
+    margin-bottom: 35px;
+  }
+
+  main section div {
+    padding-right: 10px;
+  }
+
+  main section img {
+    margin-left: -8px;
+    width: 760px;
+    height: 160px;
+  }
+
+  main .section-div-3col {
+    margin-top: -6px;
+    padding-right: 25px;
+    line-height: 1.15;
+  }
+
+  /*Footer*/
+  footer {
+    padding-left: 10px;
+    padding-top: 35px;
+  }
+
+  footer span {
+    margin-bottom: 0;
+  }
+
+  .span-big-margin {
+    margin-top: 15px;
+  }
+
+  .copyright {
+    padding-top: 40px;
+    font-size: 16px;
+  }
+}
+
+/***************500PX BREAK***********/
+
+@media (max-width: 500px) {
+  /*General*/
+  .container {
+    width: 500px;
+  }
+
+  /*Nav Area*/
+  .sticky-nav {
+    justify-content: flex-start;
+    margin-top: -10px;
+    position: relative;
+  }
+
+  .sticky-nav img {
+    padding-right: 10px;
+  }
+  nav {
+    height: 400px;
+  }
+  nav ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-content: center;
+    align-items: center;
+  }
+
+  .sticky-nav .nav-ul {
+    margin-top: 17px;
+  }
+  .nav-ul li a {
+    display: flex;
+    width: 500px;
+    height: 61px;
+    font-size: 20px;
+    padding-left: 15px;
+    align-items: center;
+    justify-content: center;
+    border-top: 1px solid black;
+  }
+
+  .nav-ul li a:hover,
+  .nav-ul li a:active {
+    background-color: black;
+    color: white;
+  }
+
+  .nav-ul li:last-child {
+    border-bottom: 1px solid black;
+  }
+
+  /*Header*/
+  .main-header {
+    border-bottom: none;
+    padding-left: 0;
+  }
+
+  .main-header:after {
+    content: "";
```

Comment:
this is throwing a weird bar in the middle of the header.

Reply:
Cannot recreate?